### PR TITLE
[FIX] hw_posbox_homepage: Fixup the homepage for the IoT Box

### PR DESCRIPTION
**Pull request by davidmonnom**
Bug fixes:
- Fix the detection of OS so if we are on windows some features are disabled
- Fix gap between top buttons
- Fix the loading screen not showing up

### CODE REVIEW

**Comment by davidmonnom** [comment]
@robodoo up to 17.0

**Review by Ysoroko — APPROVED** [review]

**Comment by Ysoroko** [comment]
@robodoo r+

File: addons/hw_posbox_homepage/static/src/app/Homepage.js [hunk 1]
```diff
@@ -82,14 +87,14 @@ export class Homepage extends Component {
...
-            <div class="position-absolute end-0 top-0 mt-3 me-4 d-flex gap-">
+            <div class="position-absolute end-0 top-0 mt-3 me-4 d-flex gap-1">
                 <IconButton onClick.bind="toggleAdvanced" icon="this.store.advanced ? 'fa-cog' : 'fa-cogs'" />
                 <IconButton onClick.bind="restartOdooService" icon="'fa-power-off'" />
             </div>
             <div class="d-flex mb-4 flex-column align-items-center justify-content-center">
                 <h4 class="text-center m-0">IoT Box - <t t-esc="this.data.hostname" /></h4>
             </div>
-            <div class="alert alert-warning" role="alert">
+            <div t-if="this.store.advanced" class="alert alert-warning" role="alert">
```
Comment (lse-odoo) [inline addons/hw_posbox_homepage/static/src/app/Homepage.js:97]: don't really like this change, we should show to the user if there is a HTTPS issue IMHO as it is unintended

**Comment by davidmonnom** [comment]
@robodoo retry

**Comment by Ysoroko** [comment]
@robodoo r+

**Pull request closed without merging**